### DISCUSSION TITLE
osu!lazer refusing to open after multiple fresh installs.

### DISCUSSION THREAD

**Discussion by HalfNathan** [ppy/osu#17022] [Q&A]
Even after multiple retries, osu!lazer does not install. Instead, when opened after downloading, it will show a osu icon on the taskbar what is performing an action. After that, it disappears. When opened manually after, it will show a waiting icon, and then just stop.

Here's a video of what is going on: https://youtu.be/ivfbMlTnVtM

**Logs not provided as they will not update with osu!lazer. They only update with the osu!stable client.**

However, it does give 4 notifications for each time I try to download and run it in event viewer:

Windows Error Reporting (1) ![wer1](https://user-images.githubusercontent.com/88352715/156084990-e4145372-fb1d-4568-8136-0f22ab62f384.PNG)
Windows Error Reporting (2) ![wer2](https://user-images.githubusercontent.com/88352715/156085021-3da8578f-fc7b-42ee-85a4-3ffb3cc8f4c6.PNG)
Application Error ![app error](https://user-images.githubusercontent.com/88352715/156085146-caf81cda-1970-4016-9fa8-c3bab2ddf072.PNG)
.NET Runtime ![net runtime](https://user-images.githubusercontent.com/88352715/156085159-80864300-c638-4275-b663-3747c690169f.PNG)

**Comment by smoogipoo** [comment]
Looks like a driver issue.

**Reply by HalfNathan** [reply]
Which drivers may I need to update? Or is this an unfixable issue?

**Reply by smoogipoo** [reply]
Try updating your GPU drivers.

**Reply by HalfNathan** [reply]
Will do.

**Reply by HalfNathan** [reply]
I have come to the conclusion that there are no GPU drivers (possibly because I play on a laptop) and that all other graphics drivers are up to date, is there any other way to fix this?

**Reply by peppy** [reply]
Can you provide more details on your hardware, specifically the GPU (or the laptop's model so we can look up the specs online)?

**Reply by smoogipoo** [reply]
Just the [dxdiag output](https://support.microsoft.com/en-us/help/4028644/windows-open-and-run-dxdiagexe) is enough.

**Reply by HalfNathan** [reply]
Sorry for the wait! Here is the dxdiag of the laptop.

![image_2022-02-28_213028](https://user-images.githubusercontent.com/88352715/156110693-8913b58c-202f-4f26-8972-5ec9181e944a.png)

**Reply by HalfNathan** [reply]
I'm pretty sure that this doesn't give any information about the laptop model, so ill just say it here.

Lenovo IdeaPad S340 with an Intel Core i3 10th gen processor.

**Reply by HalfNathan** [reply]
Just got time to send the full dxdiag. Sorry for the delay and the lack or detailed response.
[DxDiag.txt](https://github.com/ppy/osu/files/8174800/DxDiag.txt)

**Reply by bdach** [reply]
i've noticed in the dxdiag report that you seem to be using a 71Hz display mode. is this accurate? are you overclocking your monitor or something of the sort?

if yes, do you mind trying plain 60Hz and seeing if it helps anything?

**Reply by HalfNathan** [reply]
Yes, I overclock my monitor.

No, 60hz doesn't work, even after another install

**Reply by frenzibyte** [reply]
Can you attach logs of the crashes you get after running back on 60Hz, in case that's a different crash.

**Reply by HalfNathan** [reply]
Just checked event viewer, it does not show a different crash note.

**Reply by HalfNathan** [reply]
I know this thread has been dead for the longest time but I have finally gotten a proper error on this install.
![error](https://user-images.githubusercontent.com/88352715/179693490-43d57062-a2b9-4a34-a30f-8b17fccb3ec1.PNG)
I use the latest update of Windows 10. The .NET Runtime and Application Errors are all the same, not counting 'Faulting application start time' and 'Faulting application path' for the Application Error as it is different with every update.

**Reply by smoogipoo** [reply]
For that error, check right click -> properties, and somewhere there's going to be an option about running in compatibility mode. It should be (incorrectly) set to Windows 8.1.

**Reply by HalfNathan** [reply]
I guess somewhere along the way of attempting to get lazer to run on my laptop, I turned on compatibility mode because now I'm back to the same problem as in the start of this thread.

**Reply by HalfNathan** [reply]
Update: I just had the brilliant idea of unplugging my external monitor and seeing if osu!lazer would run. To my surprise, it actually did. However, when I plugged my external monitor back in and tried to put lazer in fullscreen, it immediately blue screened with the error code 'KMODE_EXCEPTION_NOT_HANDLED'

**Reply by HalfNathan** [reply]
Another update: When I mirrored my displays and ran lazer, it ended up actually working. Could it be that my two monitors are tricking the game to think that I have a single, long monitor?
